webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectExportService.scala

```diff
@@ -160,33 +176,83 @@ final case class ProjectExportServiceLive(
-      collectDir <- Files.createTempDirectoryScoped(Some(project.shortname), fileAttributes = Nil)
-      _          <- exportProjectTriples(project, trigExportFile(project, collectDir))
-      _          <- exportProjectAssets(project, collectDir)
-      zipped     <- ZipUtility.zipFolder(collectDir, exportDir)
+      exportDir    <- Files.createTempDirectory(Some(s"export-${project.shortname}"), fileAttributes = Nil)
+      collectDir   <- Files.createTempDirectoryScoped(Some(project.shortname), fileAttributes = Nil)
+      _            <- exportProjectTriples(project, trigExportFile(project, collectDir))
+      _            <- exportProjectAssets(project, collectDir, user)
+      zipped       <- ZipUtility.zipFolder(collectDir, exportDir)
+      fileSize     <- Files.size(zipped)
+      absolutePath <- zipped.toAbsolutePath
+      _            <- ZIO.logInfo(s"Exported project ${project.shortname} to $absolutePath ($fileSize bytes)")
     } yield zipped
   }
 
-  private def exportProjectAssets(project: KnoraProject, tempDir: Path) = {
+  private def exportProjectAssets(project: KnoraProject, tempDir: Path, user: UserADM): ZIO[Any, Throwable, Path] = {
     val exportedAssetsDir = tempDir / "assets"
     for {
       _ <- Files.createDirectory(exportedAssetsDir)
+      _ <- assetService.exportProjectAssets(project, exportedAssetsDir, user)
     } yield exportedAssetsDir
   }
 }
 
+trait AssetService {
+  def exportProjectAssets(project: KnoraProject, tempDir: Path, user: UserADM): Task[Path]
+}
+case class Asset(belongsToProject: KnoraProject, internalFilename: String)
+object Asset {
+  def logString(asset: Asset) = s"asset:${asset.belongsToProject.shortcode}/${asset.internalFilename}"
+}
+
+final case class AssetServiceLive(
+  private val triplestoreService: TriplestoreService,
+  private val sipiClient: IIIFService,
+  private val ontologyRepo: OntologyRepo
+) extends AssetService {
+  override def exportProjectAssets(project: KnoraProject, directory: Path, user: UserADM): Task[Path] = for {
+    _ <- ZIO.logDebug(s"Exporting assets ${project.id}")
+    assets <- determineAssets(project)
+                .tap(it => ZIO.logInfo(s"Found ${it.size} assets for project ${project.shortcode}"))
+    _ <-
+      ZIO
+        .foreachPar(assets)(sipiClient.downloadAsset(_, directory, user))
+        .withParallelism(10)
+        .tap { downloadedAssets =>
+          val nrPresent    = assets.size
+          val nrDownloaded = downloadedAssets.flatten.size
+          ZIO.logInfo(s"Successfully downloaded $nrDownloaded/$nrPresent files for project ${project.shortcode}")
+        }
+  } yield directory
+
+  private def determineAssets(project: KnoraProject): Task[List[Asset]] = {
+    val projectGraph = ProjectADMService.projectDataNamedGraphV2(project)
+    for {
+      ontologyGraphs <- ontologyRepo.findOntologyGraphsByProject(project)
+      query           = findAllAssets(ontologyGraphs :+ InternalIri(KnoraBaseOntologyIri), projectGraph)
```

Comment:
you add KnoraBase here, but no the other builtins (admin, standoff, etc.). Is this intentional or an oversight?